subsys/net/lib/lwm2m/lwm2m_engine.h

```diff
@@ -75,4 +75,11 @@ void lwm2m_udp_receive(struct net_context *ctx, struct net_pkt *pkt,
 				struct zoap_packet *response,
 				struct sockaddr *from_addr));
 
+enum zoap_block_size lwm2m_default_block_size(void);
+
+u8_t lwm2m_firmware_get_update_state(void);
```

Comment:
These functions would only be available if CONFIG_LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT is enabled.